Sources/SwiftGodotMacroLibrary/MacroGodot.swift

```diff
@@ -104,6 +104,9 @@ class GodotMacroProcessor {
         if let optSyntax = type.as (OptionalTypeSyntax.self) {
             type = optSyntax.wrappedType
         }
+        guard varDecl.isArray == false else {
+            throw GodotMacroError.requiresGArrayCollection
```

Comment:
if the `@Export` was attached to an Array type, throw this error to inform the user to use `ObjectCollection` or `VariantCollection`